closed/OpenJ9.gmk

```diff
@@ -308,6 +308,17 @@ $(foreach file, \
 	$(if $(wildcard $(file)), \
```

Comment:
Can you add the code to handle the loadLibraryTest_lib to the code just above this line?  

Something like (untested):
```
$(patsubst %, $(OPENJ9_VM_BUILD_DIR)/$(call 'lib[$]1.jnilib',%), \
            libloadLibraryTest \
            ), \
```

Reply:
I need the new `openj9_test_image_jnilib_rules` function in order to rename the library from `libloadLibraryTest.dylib` to `libloadLibraryTest.jnilib` in the destination folder. 
This is due to uma (using module.xml) unable to build shared library with a custom extension.

It might be possible to change the uma code to add a new shared library type so we can remove this extra copy target in the make file.

Reply:
I'd rather see a hack to the UMA module.xml than to patches to the Extensions repo

Reply:
I'll look into updating the UMA code then update the PR, marking this WIP until fixed

Reply:
added new target to test build script that will rename the library created in UMA